src/foam/lib/formatter/JSONFObjectFormatter.java

```diff
@@ -53,6 +54,18 @@ protected SimpleDateFormat initialValue() {
   protected boolean outputReadableDates_ = true;
   protected boolean outputDefaultClassNames_ = true;
 
+  public static ThreadLocal<FObjectFormatter> getThreadLocal(X x, Boolean quoteKeys, Boolean outputClassNames, PropertyPredicate predicate) {
```

Comment:
If this is for the HTTPBox, then why not define it in the HTTPBox class? Then everyone can have their ThradLocal JSONFObjectFormatter their way?

Reply:
I've used this 6 times in medusa and all have this same requirement.